settings/benchmarks-suite-jdk21.json

```diff
@@ -215,8 +234,10 @@
       "name": "ObjectStreamClassBenchmark"
     },
     {
-      "name": "PalindromeBenchmark",
-      "jvmArgsAppend": "-Xss20M"
```

Comment:
Same question

Reply:
Same rationale +

PalindromeBenchmark has this already declared in the class
`@Fork(value = 5, jvmArgsAppend = {"-Xss20M"})`
so it was duplicated.